fastlane/lanes/release.rb

```diff
@@ -411,6 +413,7 @@
     trigger_release_build
 
     pr_url = create_release_management_pull_request(
+      release_version: release_version_next,
       base_branch: DEFAULT_BRANCH,
       title: "Merge #{version} release finalization"
```

Comment:
I don't know if it's documented, or even agreed upon, anywhere, but I've historically set the release finalization as a PR in the next release milestone. The rationale being that it's more valuable as a requirement for the next release more than as a PR in the release milestone.

Practically, this helped me notice when the finalization PR hadn't been merged. Now that a bot makes the PR and the release manager can review and merge it, maybe there's less value in this approach.

Let me know what you think. Here's how it would look like with the current release version

```suggestion
      release_version: version,
      base_branch: DEFAULT_BRANCH,
      title: "Merge #{version} release finalization"
```

Reply:
Yeah I remember us having this conversation somewhere, though I don't remember if it was on Slack or PR, and I don't think it was officialized in P2?

Anyway, I agree with the rationale.

FYI, in Tumblr we simply don't set a milestone at all for release-related PRs (and Danger is configured to not warn about that if PR has `tooling` or `release` label). But our Fastfile creates those PR using the bot user (which thus becomes the author of those PRs) and adds the Release Manager as reviewer, so we're less likely to miss them; our release scenario also reminds us to merge those PRs in time before the next step too.

That being said, maybe we should follow your footsteps and add a milestone to those (also because it make sense as a signal/documentation/flagging, as those PRs indeed have a deadline they need to be merged before…)